### PR TITLE
lastDrugs

### PR DESCRIPTION
**Pull request by Guerschon**

### CODE REVIEW

File: sqldiff.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sqldiff PUBLIC "-//OpenMRS//DTD OpenMRS SQL Diff Config 1.0//EN" "http://resources.openmrs.org/doctype/sqldiff-1.0.dtd">
+
```
Comment (k-joseph) [inline sqldiff.xml]: this file is in the wrong place, what happened to your repo to have these files misplaced

File: omod/src/main/webapp/fragments/lastDrugs.gsp [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class="info-section">
```
Comment (k-joseph) [inline omod/src/main/webapp/fragments/lastDrugs.gsp]: this is part of all i need for this commit

File: omod/src/main/webapp/fragments/isantePlusManager.gsp [hunk 3]
```diff
@@ -2,6 +2,7 @@
 <input type="checkbox" class="managerOption" id="${manager.patientFormHistoryExtensionId}" ${manager.patientFormHistoryExtensionChecked}> ${ ui.message('isanteplus.manageIsantePlusPatientDashboard.enablePatientFormHistory') }<br />
 <input type="checkbox" class="managerOption" id="${manager.growthChartsExtensionId}" ${manager.growthChartsExtensionChecked}> ${ ui.message('isanteplus.manageIsantePlusPatientDashboard.enableGrowthCharts') }<br />
 <input type="checkbox" class="managerOption" id="${manager.lastViralLoadTestExtensionId}" ${manager.lastViralLoadTestExtensionChecked}> ${ ui.message('isanteplus.manageIsantePlusPatientDashboard.enableLastViralLoadTest') }<br />
+<input type="checkbox" class="managerOption" id="${manager.lastDrugsExtensionId}" ${manager.lastDrugsExtensionChecked}> ${ ui.message('isanteplus.manageIsantePlusPatientDashboard.enableLastDrugs') }<br />
```
Comment (k-joseph) [inline omod/src/main/webapp/fragments/isantePlusManager.gsp]: this is part of all i need for this commit

File: omod/src/main/resources/apps/isantepluspatientdashboard_extension.json [hunk 4]
```diff
@@ -8,6 +8,14 @@
         }
     },
     {
+        "id": "isanteplus.patientDashboard.secondColumnFragments.lastDrugs",
+        "extensionPointId": "patientDashboard.secondColumnFragments",
```
Comment (k-joseph) [inline omod/src/main/resources/apps/isantepluspatientdashboard_extension.json]: this is part of all i need for this commit

File: omod/src/main/java/org/openmrs/module/isanteplus/fragment/controller/LastDrugsFragmentController.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
+package org.openmrs.module.isanteplus.fragment.controller;
+
+
+import java.util.List;
+
+import org.apache.commons.logging.Log;
```
Comment (k-joseph) [inline omod/src/main/java/org/openmrs/module/isanteplus/fragment/controller/LastDrugsFragmentController.java]: this is part of all i need for this commit

File: moduleApplicationContext.xml [hunk 6]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (k-joseph) [inline moduleApplicationContext.xml]: this file is in the wrong place, what happened to your repo to have these files misplaced

File: messages_fr.properties [hunk 7]
```diff
@@ -0,0 +1,91 @@
+isanteplus.title=iSantePlus Module
```
Comment (k-joseph) [inline messages_fr.properties]: this file is in the wrong place, what happened to your repo to have these files misplaced

File: messages.properties [hunk 8]
```diff
@@ -0,0 +1,95 @@
+isanteplus.title=iSantePlus Module
```
Comment (k-joseph) [inline messages.properties]: this file is in the wrong place, what happened to your repo to have these files misplaced

File: liquibase.xml [hunk 9]
```diff
@@ -0,0 +1,292 @@
...
+				appframework_component_state WHERE component_id =
+				'org.openmrs.module.isantepluspatientdashboard.patientDashboard.secondColumnFragments.weightGraph';
+			</sqlCheck>
+		</preConditions>
+		<comment></comment>
+		<insert tableName="appframework_component_state">
+			<column name="component_id"
+				value="org.openmrs.module.isantepluspatientdashboard.patientDashboard.secondColumnFragments.weightGraph" />
+			<column name="enabled" value="1" />
+			<column name="component_type" value="EXTENSION" />
```
Comment (k-joseph) [inline liquibase.xml]: this file is in the wrong place, what happened to your repo to have these files misplaced

File: api/src/main/java/org/openmrs/module/isanteplus/IsantePlusManager.java [hunk 10]
```diff
@@ -31,6 +34,8 @@
 	private boolean toggleGrowthChartsExtension;
 
 	private boolean toggleLastViralLoadTestExtension;
+	
+	private boolean toggleLastDrugsExtension;
```
Comment (k-joseph) [inline api/src/main/java/org/openmrs/module/isanteplus/IsantePlusManager.java]: these changes are part of what all i need for this commit

File: META-INF/maven/org.openmrs.module/isanteplus-api/pom.xml [hunk 11]
```diff
@@ -0,0 +1,92 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (k-joseph) [inline META-INF/maven/org.openmrs.module/isanteplus-api/pom.xml]: this file is in the wrong place, what happened to your repo to have these files misplaced

File: IsantePlus.hbm.xml [hunk 12]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<!DOCTYPE hibernate-mapping PUBLIC
```
Comment (k-joseph) [inline IsantePlus.hbm.xml]: this file is in the wrong place, what happened to your repo to have these files misplaced

File: META-INF/MANIFEST.MF [hunk 13]
```diff
@@ -0,0 +1,6 @@
+Manifest-Version: 1.0
```
Comment (k-joseph) [inline META-INF/MANIFEST.MF]: this file is in the wrong place, what happened to your repo to have these files misplaced

File: META-INF/maven/org.openmrs.module/isanteplus-api/pom.properties [hunk 14]
```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
+#Thu Sep 29 10:21:37 CST 2016
+version=1.0-SNAPSHOT
+groupId=org.openmrs.module
+artifactId=isanteplus-api
```
Comment (k-joseph) [inline META-INF/maven/org.openmrs.module/isanteplus-api/pom.properties]: this file is in the wrong place, what happened to your repo to have these files misplaced

**Comment by Guerschon** [comment]
We devlop on Windows


Envoyé depuis mon smartphone Samsung Galaxy.
-------- Message d'origine --------De : k-joseph <notifications@github.com> Date : 29/09/2016  4:13 PM  (GMT-05:00) À : IsantePlus/openmrs-module-isanteplus <openmrs-module-isanteplus@noreply.github.com> Cc : Guerschon <francois_guerschon@yahoo.fr>, Author <author@noreply.github.com> Objet : Re: [IsantePlus/openmrs-module-isanteplus] lastDrugs 
@k-joseph commented on this pull request.

Please clear up these problems before i can re-view once again

In sqldiff.xml:
> @@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sqldiff PUBLIC "-//OpenMRS//DTD OpenMRS SQL Diff Config 1.0//EN" "http://resources.openmrs.org/doctype/sqldiff-1.0.dtd">
+

this file is in the wrong place, what happened to your repo to have these files misplaced



In omod/src/main/webapp/fragments/lastDrugs.gsp:
> @@ -0,0 +1,14 @@
+<div class="info-section">

this is part of all i need for this commit



In omod/src/main/webapp/fragments/isantePlusManager.gsp:
> @@ -2,6 +2,7 @@
 <input type="checkbox" class="managerOption" id="${manager.patientFormHistoryExtensionId}" ${manager.patientFormHistoryExtensionChecked}> ${ ui.message('isanteplus.manageIsantePlusPatientDashboard.enablePatientFormHistory') }<br />
 <input type="checkbox" class="managerOption" id="${manager.growthChartsExtensionId}" ${manager.growthChartsExtensionChecked}> ${ ui.message('isanteplus.manageIsantePlusPatientDashboard.enableGrowthCharts') }<br />
 <input type="checkbox" class="managerOption" id="${manager.lastViralLoadTestExtensionId}" ${manager.lastViralLoadTestExtensionChecked}> ${ ui.message('isanteplus.manageIsantePlusPatientDashboard.enableLastViralLoadTest') }<br />
+<input type="checkbox" class="managerOption" id="${manager.lastDrugsExtensionId}" ${manager.lastDrugsExtensionChecked}> ${ ui.message('isanteplus.manageIsantePlusPatientDashboard.enableLastDrugs') }<br />

this is part of all i need for this commit



In omod/src/main/resources/apps/isantepluspatientdashboard_extension.json:
> @@ -8,6 +8,14 @@
         }
     },
     {
+        "id": "isanteplus.patientDashboard.secondColumnFragments.lastDrugs",
+        "extensionPointId": "patientDashboard.secondColumnFragments",

this is part of all i need for this commit



In omod/src/main/java/org/openmrs/module/isanteplus/fragment/controller/LastDrugsFragmentController.java:
> @@ -0,0 +1,23 @@
+package org.openmrs.module.isanteplus.fragment.controller;
+
+
+import java.util.List;
+
+import org.apache.commons.logging.Log;

this is part of all i need for this commit 



In moduleApplicationContext.xml:
> @@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>

this file is in the wrong place, what happened to your repo to have these files misplaced



In messages_fr.properties:
> @@ -0,0 +1,91 @@
+isanteplus.title=iSantePlus Module

this file is in the wrong place, what happened to your repo to have these files misplaced



In messages.properties:
> @@ -0,0 +1,95 @@
+isanteplus.title=iSantePlus Module

this file is in the wrong place, what happened to your repo to have these files misplaced



In liquibase.xml:
> +		</insert>
+	</changeSet>
+	<changeSet id="2016Sept02-1529" author="k-joseph">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*) FROM
+				appframework_component_state WHERE component_id =
+				'org.openmrs.module.isantepluspatientdashboard.patientDashboard.secondColumnFragments.weightGraph';
+			</sqlCheck>
+		</preConditions>
+		<comment></comment>
+		<insert tableName="appframework_component_state">
+			<column name="component_id"
+				value="org.openmrs.module.isantepluspatientdashboard.patientDashboard.secondColumnFragments.weightGraph" />
+			<column name="enabled" value="1" />
+			<column name="component_type" value="EXTENSION" />

this file is in the wrong place, what happened to your repo to have these files misplaced



In api/src/main/java/org/openmrs/module/isanteplus/IsantePlusManager.java:
> @@ -31,6 +34,8 @@
 	private boolean toggleGrowthChartsExtension;
 
 	private boolean toggleLastViralLoadTestExtension;
+	
+	private boolean toggleLastDrugsExtension;

these changes are part of what all i need for this commit



In META-INF/maven/org.openmrs.module/isanteplus-api/pom.xml:
> @@ -0,0 +1,92 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

this file is in the wrong place, what happened to your repo to have these files misplaced



In IsantePlus.hbm.xml:
> @@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<!DOCTYPE hibernate-mapping PUBLIC

this file is in the wrong place, what happened to your repo to have these files misplaced



In META-INF/MANIFEST.MF:
> @@ -0,0 +1,6 @@
+Manifest-Version: 1.0

this file is in the wrong place, what happened to your repo to have these files misplaced



In META-INF/maven/org.openmrs.module/isanteplus-api/pom.properties:
> @@ -0,0 +1,5 @@
+#Generated by Maven
+#Thu Sep 29 10:21:37 CST 2016
+version=1.0-SNAPSHOT
+groupId=org.openmrs.module
+artifactId=isanteplus-api

this file is in the wrong place, what happened to your repo to have these files misplaced

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/IsantePlus/openmrs-module-isanteplus","title":"IsantePlus/openmrs-module-isanteplus","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/IsantePlus/openmrs-module-isanteplus"}},"updates":{"snippets":[{"icon":"PERSON","message":"@k-joseph commented on #6"}],"action":{"name":"View Pull Request","url":"https://github.com/IsantePlus/openmrs-module-isanteplus/pull/6#pullrequestreview-2231444"}}}

**Comment by k-joseph** [comment]
then clear up things as i mentioned

**Comment by k-joseph** [comment]
do that in a new commit

**Pull request closed without merging**